libbpf-cargo/src/build.rs

```diff
@@ -174,6 +174,8 @@ fn compile_one(debug: bool, source: &Path, out: &Path, clang: &Path, options: &s
         let arch = match ARCH {
             "x86_64" => "x86",
             "aarch64" => "arm64",
+            "powerpc64" => "powerpc",
```

Comment:
Looking at libbpf-bootstrap's transformation (https://github.com/libbpf/libbpf-bootstrap/blob/master/examples/c/Makefile#L12C1-L18C38):

ARCH ?= $(shell uname -m | sed 's/x86_64/x86/' \
			 | sed 's/arm.*/arm/' \
			 | sed 's/aarch64/arm64/' \
			 | sed 's/ppc64le/powerpc/' \
			 | sed 's/mips.*/mips/' \
			 | sed 's/riscv64/riscv/' \
			 | sed 's/loongarch64/loongarch/')

It seems like we expect ppc64le for powerpc. Can you confirm that it actually is "powerpc64" and not "ppc64le"?

Seems like we don't normalize s390x to s390 in libbpf-bootstrap either, but probably no one ever bothered to build libbpf-bootstrap on s390 :)

Reply:
I got the error when [building](https://koji.fedoraproject.org/koji/taskinfo?taskID=110351127) a project using libbpf-cargo on a Fedora package builder for ppc64le, so I can confirm that `ARCH` is set to `powerpc64`.

The documentation of [`std::env::consts::ARCH`](https://doc.rust-lang.org/std/env/consts/constant.ARCH.html) also mentions powerpc64 but not ppc64le.

Reply:
> The documentation of [`std::env::consts::ARCH`](https://doc.rust-lang.org/std/env/consts/constant.ARCH.html) also mentions powerpc64 but not ppc64le.

Yeah, I think that is the authoritative source for the input to the `match` expression. `uname -m` is free to produce different values.